addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -0,0 +1,54 @@
+import { merge } from '@ember/polyfills';
+import getElement from './-get-element';
+import triggerEvent from './trigger-event';
+import { KEYBOARD_EVENT_TYPES } from './fire-event';
+
+const DEFAULT_MODIFIERS = Object.freeze({
+  ctrlKey: false,
+  altKey: false,
+  shiftKey: false,
+  metaKey: false,
+});
+
+/**
+  @public
+  @param {String|HTMLElement} target
+  @param {'keydown' | 'keyup' | 'keypress'} eventType
+  @param {String} keyCode
+  @param {Object} [modifiers]
+  @param {Boolean} [modifiers.ctrlKey=false]
+  @param {Boolean} [modifiers.altKey=false]
+  @param {Boolean} [modifiers.shiftKey=false]
+  @param {Boolean} [modifiers.metaKey=false]
+  @return {Promise<void>}
+*/
+export default function triggerKeyEvent(target, eventType, keyCode, modifiers = DEFAULT_MODIFIERS) {
```

Comment:
I've noticed a name change here. In ember-native-dom-helpers it's named just `keyEvent`, which I copied from the current acceptance helper: https://emberjs.com/api/ember/2.15/classes/Ember.Test/methods/keyEvent?anchor=keyEvent&show=inherited%2Cprotected%2Cprivate%2Cdeprecated

I don't oppose to the name change, but it won't be a drop-in replacement for ember-native-dom-helpers.

Reply:
Aye, agreed.  The change was intentional (`keyEvent` "seemed" odd to me when what you are doing is triggering a keyboard event).

I think we can mitigate or at least ease some of the migration concerns by adding a reexport to `ember-native-dom-helpers`...